backend/src/helpers/token/create-token/create-token.helper.ts

```diff
@@ -0,0 +1,7 @@
+import jwt, { Secret } from 'jsonwebtoken';
+import { ENV } from '~/common/enums/app/env.enum';
+
+const createToken = (data: string): string =>
```

Comment:
we also need service for token. in the future we will have `verify` and other methods